packages/patternfly-3/react-console/src/VncConsole/VncActions.js

```diff
@@ -1,6 +1,10 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import { DropdownButton, MenuItem, noop } from 'patternfly-react';
+import { MenuItem } from 'patternfly-react';
+
+// FIXME: DropdownButton is inaccessible after 1f3411a6e708e833e48921380ae736cf5f79862b commit
+import DropdownButton from 'patternfly-react/dist/esm/components/Button/DropdownButton';
```

Comment:
@mareklibra I will look into this.

Reply:
Yeah... so this was broken for the patternfly-react packages by #853 like you say.
Unfortunately, I can think of only 2 fixes:

1) Create directories for all subcomponents and move them there. This would, however, be a breaking change externally as anyone doing path imports would no longer find the subcomponent in the same location.

2) Add subcomponents to the main component and reference them through that (only required in patternfly-react packages not externally). That is, Button would have Button.Dropdown and you would import Button and use that . Similar to Row and Col in Grid.  

Can you think of any other options?